### ISSUE TITLE
Suggestion: move `sentry` and `HerokuWorker` to a new `extensions` modules

### ISSUE THREAD

**Issue by lowercase00** [rq/rq#1861]
Both `sentry` and `Heroku` are third party and not inherently related to `rq`.
As much as both are relevant (and should continue to be for the foreseeable future I guess), my suggestion is to move both to a dedicated `extensions` module. This would allow for a greater separation of concerns (third party vs core codebase) and easier contribution from third party maintainers.

**Comment by selwin** [comment]
Agreed, we already have a `contrib` module and the newer Sentry integration already lives there.

Further cleanups are welcome :)
